curso/semana_1/sesion_1/Opcionales.playground/Contents.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+
+do {
+    // userFavoriteAnimal se asigna nil por default
+    var userFavoriteAnimal = Optional<String>("Dogs")
```

Comment:
Como lo comento Anber, esta forma fácilmente podría omitirse por:

```suggestion
    var userFavoriteAnimal: String? = "Dogs"
```